platform/features/listview/res/templates/listview.html

```diff
@@ -0,0 +1,58 @@
+<div ng-controller="ListViewController">
+  <table>
+    <!-- Table Header -->
+    <tr>
+      <th>Icon</th>
+
+      <th>
+        <a href="" ng-click="orderByField='title'; reverseSort = !reverseSort">
+          Name
+          <span ng-show="orderByField == 'title'">
+            <span ng-show="!reverseSort">^</span><span ng-show="reverseSort">v</span>
+          </span>
+        </a>
+      </th>
+
+      <th>
+        <a href="" ng-click="orderByField='type'; reverseSort = !reverseSort">
+          Type
+          <span ng-show="orderByField == 'type'">
+            <span ng-show="!reverseSort">^</span><span ng-show="reverseSort">v</span>
+          </span>
+        </a>
+      </th>
+
+      <th>
+        <a href="" ng-click="orderByField='persisted'; reverseSort = !reverseSort">
+          Created Date
+          <span ng-show="orderByField == 'persisted'">
+            <span ng-show="!reverseSort">^</span><span ng-show="reverseSort">v</span>
+          </span>
+        </a>
+      </th>
+
+      <th>
+        <a href="" ng-click="orderByField='modified'; reverseSort = !reverseSort">
+          Update Date
+          <span ng-show="orderByField == 'modified'">
+            <span ng-show="!reverseSort">^</span><span ng-show="reverseSort">v</span>
+          </span>
+        </a>
+      </th>
+    </tr>
+
+    <!-- Table Contents -->
+    <tr ng-repeat="composition in composees|orderBy:orderByField:reverseSort"
+        mct-gesture="menu,info">
+      <th>
```

Comment:
THAT'S why your table rows weren't looking right to me. Your table markup isn't quite right yet, notably, you're using `<th>` elements in the body of the table instead of `<td>`'s. Also, you don't have to, but it's not a bad practice to use `<thead>` and `<tbody>`. So:

~~~~
<table>
  <thead>
    <tr>
      <th>Header Title</th><th>Etc.</th>
    </tr>
  </thead>
  <tbody>
    <tr repeat as needed>
      <td>Item</td><th>Etc.</th>
    <tr>
  </tbody>
</table>